packages/react-swipeable-views/src/SwipeableViews.js

```diff
@@ -369,6 +369,10 @@ class SwipeableViews extends Component {
      * If the computed speed is above this value, the index change.
      */
     threshold: PropTypes.number,
+    /**
+     * This is the config used to disable lazyloding
+     */
+    lazyLoading: PropTypes.bool,
```

Comment:
Can we use the following wording: `disableLazyLoading`. 
The name of the boolean properties should be chosen based on the default value. We are following the HTML specification. For instance, the disabled attribute on an input element. This choice allows the shorthand notation.